src/garvis/capability_runtime.py

```
@@ -98,6 +117,76 @@ def __init__(
+                        ),
+                        "council_participation_count": (
+                            report.council_participation_count
+                        ),
+                        "angel_participation_count": (
+                            report.angel_participation_count
+                        ),
+                        "operational_authorization": False,
+                    },
+                )
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)